src/store/modules/activeRows/types.ts

```diff
@@ -0,0 +1,26 @@
+/*----------------------- State -----------------------*/
+/** currentFarmer State*/
+export interface ActiveRowsState {
```

Comment:
why have you renamed it this? Do you plan to put all active info in this like the milk transaction, trader and farmer?